.github/workflows/tests.yml

```diff
@@ -123,6 +123,12 @@ jobs:
             python-version: "3.10"
             pytest-extra-options: "-W ignore::ImportWarning -k 'not (test_correlation or test_interpolate or test_mcsolve)'"
 
+          # Builds without Cython at runtime.  This is a core feature;
+          # everything should be able to run this.
```

Comment:
We should update the comment to mention why we have an extra non-Cython run for 3.12.